app/views/customer_service/index.html.erb

```diff
@@ -0,0 +1,332 @@
+<%- @page_title = 'Customer Service' -%>
+
+<!-- Don't automatically turn FAX numbers into links -->
+<meta name = "format-detection" content = "telephone=no">
+
+<div class="usa-grid usa-section usa-layout-docs">
+  <%= render 'shared/sidenav' %>
+
+  <div class="usa-width-three-fourths usa-layout-docs-main_content">
+    <h1><%= @page_title %></h1>
+
+    <div class="usa-grid-full customer-service-section" id="local-ppo">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("pppo.png") %>
```

Comment:
FYI, I just checked, and rails used the original file name when inferring the alt text: 

```<img src="/assets/svc-help-74fdd0de9024d65d1dcc37f169e45a85e8e620d6d94c4ee7e81670346f9cbce6.png" alt="Svc help">```